test/e2e/providers_test.go

```diff
@@ -152,7 +139,12 @@ func setOperatorConfigStatuspageIDProvider(t *testing.T, client *testframework.C
 			},
 		},
 	}
-	_, err = client.Consoles().Update(operatorConfig)
+
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
👍

Reply:
I bet there are a few more places that could use this Retry wrapper.